nym-vpn-desktop/src-tauri/src/network.rs

```diff
@@ -1,9 +1,62 @@
-use tracing::debug;
+use std::path::PathBuf;
+
+use anyhow::{anyhow, Context, Result};
+use nym_vpn_lib::nym_config;
+use tokio::fs::{read, try_exists};
+use tracing::{debug, error, info, instrument};
+
+#[instrument]
+pub async fn setup_network_env(use_sandbox: bool, env_config_file: &Option<PathBuf>) -> Result<()> {
+    if use_sandbox {
+        info!("network environment: sandbox");
+        // TODO: instead bundle a sandbox env file we can use
+        setup_sandbox_environment();
+        return Ok(());
```

Comment:
Style suggestion: it feels more rust to do
```
return Ok(setup_sandbox_environment());
```
since it's more expression oriented than statement oriented

Reply:
![image](https://github.com/nymtech/nym-vpn-client/assets/6359431/425e5975-fc93-43a5-b1b9-8929208a59cd)